FirebaseStorage/Tests/Unit/StorageTests.swift

```diff
@@ -239,6 +247,15 @@ class StorageTests: XCTestCase {
     XCTAssertEqual(32.0, Storage.computeRetryInterval(fromRetryTime: 60.0))
   }
 
+  func testRetryTimeChange() throws {
```

Comment:
Add the always missing test for implicit interval changes based on retrytime changes.